src/Core/src/Platform/Android/MauiSwipeView.cs

```diff
@@ -665,6 +665,12 @@ List<AView> GetNativeSwipeItems()
 
 		void UpdateSwipeItemViewLayout(ISwipeItemView swipeItemView)
 		{
+			if (swipeItemView?.Handler is not IPlatformViewHandler handler)
+				return;
+
+			var swipeItemSize = GetSwipeItemSize(swipeItemView);
+			handler.LayoutVirtualView(0, 0, (int)swipeItemSize.Width, (int)swipeItemSize.Height);
```

Comment:
This is the main change. 

`View.Arrange(new Rectangle(Point.Zero, size));`

Doesn't seem to work as expected.